portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,26 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private List<String> myMessages;
+
+    @Override
+    public void init() {
+        myMessages = new ArrayList<>();
+        myMessages.add("Hello my name is Sofia Fernandez and I am a CS major at Georgia Tech.");
+        myMessages.add("This is my first time working with servlets, so I am just testing this.");
+        myMessages.add("This is the third hard-coded message that should be added to my arraylist.");
+    }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<p>Hello Sofia!<p>");
+        String myJson = convertToJsonWithGson(myMessages);
+        response.setContentType("application/json;");
+        response.getWriter().println(myJson);
+  }
+
+  private String convertToJsonWithGson(List<String> arraylist) {
```

Comment:
This doesn't seem to depend on the member variables. If so, make it static?

Reply:
You are right, added!